tests/spec/fs.stat.spec.js

```diff
@@ -92,3 +92,24 @@ describe('fs.stat', function() {
     });
   });
 });
+
+/**
+ * fsPromises tests
+ */
+
+describe('fsPromises.stat', function() {
+  beforeEach(util.setup);
+  afterEach(util.cleanup);
+
+  it('should return an error if path does not exist', function(done) {
+    var fsPromises = util.fs().promises;
+
+    fsPromises.stat('/tmp')
```

Comment:
You should use `return` as `return fsPromises.stat('/tmp')

Reply:
Logically, there is no error on 9ceae9b76193531d1f7625a211d267961ec6b31f. It is just the tester will wait until the promise chain fires `done()` to continue the next tests. The common error is to return promise while also calling `done()`, it will throw an exception as stated on https://mochajs.org/#working-with-promises.

However your suggestion should be the right way to test synchronous codes. Thank you for the nice review, I'll fix as you suggested.